.agents/skills/stellar-dev/testing.md

```
@@ -0,0 +1,795 @@
+          stellar keys generate --global ci-test --network local --fund
+          stellar contract deploy \
+            --wasm target/wasm32-unknown-unknown/release/contract.wasm \
+            --source ci-test \
+            --network local
+```
+
+## Best Practices
+
+### Test Organization
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)